presto-main/src/main/java/com/facebook/presto/operator/repartition/Int128ArrayBlockEncodingBuffer.java

```diff
@@ -109,6 +109,15 @@ public long getSerializedSizeInBytes()
                 getNullsBufferSerializedSizeInBytes();    // nulls buffer
     }
 
+    @Override
+    public String toString()
+    {
+        StringBuilder sb = new StringBuilder("Int128ArrayBlockEncodingBuffer{");
```

Comment:
better Int128ArrayBlockEncodingBuffer.class.getSimpleName(); ditto other places

Reply:
Sure. Changed to
```
StringBuilder sb = new StringBuilder(getClass().getSimpleName()).append("{");
```